backup.diff

```diff
@@ -0,0 +1,145 @@
+index 676b04d3fe..024d3d4e21 100644
+--- a/hbase-it/src/test/java/org/apache/hadoop/hbase/chaos/factories/ServerAndDependenciesKillingMonkeyFactory.java
++++ b/hbase-it/src/test/java/org/apache/hadoop/hbase/chaos/factories/ServerAndDependenciesKillingMonkeyFactory.java
+@@ -50,16 +50,16 @@ public class ServerAndDependenciesKillingMonkeyFactory extends MonkeyFactory {
+ 
+     // Destructive actions to mess things around. Cannot run batch restart.
+     Action[] actions1 = new Action[]{
+-      new RestartRandomRsExceptMetaAction(60000),
+-      new RestartActiveMasterAction(5000),
+-      new RollingBatchRestartRsAction(5000, 1.0f, 2,
+-        true), // only allow 2 servers to be dead.
+-      new ForceBalancerAction(),
+-      new RestartRandomDataNodeAction(60000),
+-      new RestartRandomZKNodeAction(60000),
++      //new RestartRandomRsExceptMetaAction(60000),
++      //new RestartActiveMasterAction(5000),
++      //new RollingBatchRestartRsAction(5000, 1.0f, 2,
++      //  true), // only allow 2 servers to be dead.
++      //new ForceBalancerAction(),
++      //new RestartRandomDataNodeAction(60000),
++      //new RestartRandomZKNodeAction(60000),
+       new GracefulRollingRestartRsAction(gracefulRollingRestartTSSLeepTime),
+-      new RollingBatchSuspendResumeRsAction(rollingBatchSuspendRSSleepTime,
+-          rollingBatchSuspendtRSRatio)
++      //new RollingBatchSuspendResumeRsAction(rollingBatchSuspendRSSleepTime,
++      //    rollingBatchSuspendtRSRatio)
+     };
+ 
+     // Action to log more info for debugging
+diff --git a/hbase-server/src/test/java/org/apache/hadoop/hbase/client/TestScannersFromClientSide.java b/hbase-server/src/test/java/org/apache/hadoop/hbase/client/TestScannersFromClientSide.java
+index e10f34257e..8afca2447b 100644
+--- a/hbase-server/src/test/java/org/apache/hadoop/hbase/client/TestScannersFromClientSide.java
++++ b/hbase-server/src/test/java/org/apache/hadoop/hbase/client/TestScannersFromClientSide.java
+@@ -94,6 +94,8 @@ public class TestScannersFromClientSide {
+   private static byte [] ROW = Bytes.toBytes("testRow");
+   private static byte [] FAMILY = Bytes.toBytes("testFamily");
+   private static byte [] QUALIFIER = Bytes.toBytes("testQualifier");
++  private static byte [] QUALIFIER1 = Bytes.toBytes("testQualifier1");
++  private static byte [] QUALIFIER2 = Bytes.toBytes("testQualifier2");
+   private static byte [] VALUE = Bytes.toBytes("testValue");
+ 
+   @Rule public TableNameTestRule name = new TableNameTestRule();
+@@ -120,8 +122,8 @@ public class TestScannersFromClientSide {
+   public static Collection<Object[]> parameters() {
+     return Arrays.asList(new Object[][] {
+         { MasterRegistry.class, 1},
+-        { MasterRegistry.class, 2},
+-        { ZKConnectionRegistry.class, 1}
++        //{ MasterRegistry.class, 2},
++        //{ ZKConnectionRegistry.class, 1}
```

Comment:
Same as above for other block of commented code.

Reply:
Yeah, same as above.